workers_kv.go

```diff
@@ -237,6 +237,26 @@ func (api API) DeleteWorkersKV(ctx context.Context, namespaceID, key string) (Re
 	return result, err
 }
 
+// DeleteWorkersKVBulk deletes multiple KVs at once.
+//
+// API reference: https://api.cloudflare.com/#workers-kv-namespace-delete-multiple-key-value-pairs
+func (api *API) DeleteWorkersKVBulk(ctx context.Context, namespaceID string, keys []string) (Response, error) {
+	uri := fmt.Sprintf("/accounts/%s/storage/kv/namespaces/%s/bulk", api.AccountID, namespaceID)
+	res, err := api.makeRequestWithAuthTypeAndHeaders(
```

Comment:
Any reason we're not using one of the wrapper methods here such as [`makeRequestWithHeaders`](https://github.com/cloudflare/cloudflare-go/blob/master/cloudflare.go#L175)?

Reply:
I was following the same format as the function [WriteWorkersKVBulk](https://github.com/cloudflare/cloudflare-go/blob/master/workers_kv.go#L194) in the same file. If the context `ctx` here does not need to be passed, I'm happy to use `makeRequestWithHeaders`. In this case, I can update `WriteWorkersKVBulk` as well.

Reply:
Just to follow up, would you like me to use `makeRequestWithHeaders`? 😄

Reply:
Yes, please.

Reply:
Updated `WriteWorkersKV`, `WriteWorkersKVBulk`, `DeleteWorkersKVBulk` 👍